changelog/62179.fixed

```diff
@@ -0,0 +1 @@
+Fixed issue with forward slashes on Windows with file.recurse and clean=True
```

Comment:
The issue number is 62197. Can you change the filename to be `62197.fixed`

Reply:
I don't know what's wrong with me and changelog files